### PR TITLE
feat: replay intro animation for activity 1

### PR DESCRIPTION
**Pull request by darren2hang**
- I added CarouselContext to activity 1
- I made the reloadTime as keys to the Convo components
- I also gave Slide3 an animation time
closes: #63

### CODE REVIEW

File: src/components/Activities/Activity1/index.tsx [hunk 1]
```diff
@@ -52,22 +52,23 @@ function Activity1(): JSX.Element {
   ];
 
   const Slide3 = () => {
+    const {reloadTime} = useContext(CarouselContext);
```
Comment (darren2hang) [inline src/components/Activities/Activity1/index.tsx:55]: should this be in the Activty1 function instead of inside Slide3?

Comment (reginawang99) [reply src/components/Activities/Activity1/index.tsx:55]: i think this is fine here for now! i think at this point tho, maybe this component should be put into another file? with a more descriptive name.

i think in the future, we might refactor and bring the reloadTime out to become props, but for now, i think its fine

**Review by reginawang99 — CHANGES_REQUESTED** [review]
tiny dot

File: src/components/Activities/Activity1/PersonCompConvoAnimation.tsx [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+import { CarouselContext } from '../../shared/Carousel';
+import { TextBubbleStyles } from '../../shared/PlaynetConstants';
+import Convo, { Phrase } from './Convo';
+
+
+interface PersonCompConvoAnimationProps {
+  timeBtwnWords: number;
+}
+
+const PersonCompConvoAnimation = (props: PersonCompConvoAnimationProps): JSX.Element => {
```
Comment (reginawang99) [inline src/components/Activities/Activity1/PersonCompConvoAnimation.tsx:18]: nit: name is a tad long
```
```suggestion
const ComputerConversation= (props: ComputerConversationProps): JSX.Element => {
```

**Review by reginawang99 — CHANGES_REQUESTED** [review]
another dot

File: src/components/Activities/Activity1/PersonCompConvoAnimation.tsx (hunk 2 above)
Comment (darren2hang) [reply src/components/Activities/Activity1/PersonCompConvoAnimation.tsx]: ok

**Review by reginawang99 — APPROVED** [review]
nice approved!

**Pull request merged**